src/mixins/transparentize.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import { isRgb, isRgba, isHsl, isHsla, isHex } from '../internalHelpers/_isValidColor'
+import hexToRgb from '../internalHelpers/_hexToRgb'
+
+const getNewTransparency = (currentTransparency: number, percentage: number) : number => (
+  currentTransparency * (1 - (percentage / 100))
+)
+
+const getColorValuesInBrackets = (color: string) : string => (
+  color.slice(color.indexOf('(') + 1, color.indexOf(')'))
+)
+
+const getValuesTillLastComma = (color: string) : string => (
+  color.slice(0, color.lastIndexOf(','))
+)
+
+const transparentizeRgb = (rgb: string, percentage: number) : string => (
+  `rgba(${getColorValuesInBrackets(rgb)}, ${getNewTransparency(1, percentage)})`
+)
+
+const getLastColorValue = (color: string) : number => (
+  Number(
+    color.slice(
+      color.lastIndexOf(',') + 1,
+      color.indexOf(')'),
+    ),
+  )
+)
+
```

Comment:
Can you add a comment like these? https://github.com/styled-components/polished/blob/master/src/helpers/hsla.js#L7-L32

We use it to generate the docs.